bundles/org.openhab.binding.livisismarthome/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -389,4 +389,12 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<!-- RebootAction -->
+	<channel-type id="restart">
```

Comment:
This should probably be marked as advanced:
```suggestion
	<channel-type id="restart" advanced="true">
```

Reply:
done